src/Xmf2.NavigationGraph.iOS/NavigationStack.cs

```diff
@@ -16,12 +16,15 @@ public class NavigationStack<TViewModel> where TViewModel : IViewModel
 
 		public void EnsureInitialized(UIWindow window)
 		{
-			//TODO POURRAIT ÊTRE MIEUX FAIT
```

Comment:
no way, revert !!!!!!

Reply:
Previous version does not work neither with the sample app, nor with the one I'm working on.